pkg/api/vlabs/validate.go

```diff
@@ -700,6 +700,9 @@ func (a *Properties) validateAddons() error {
 					if a.HasCoreOS() {
 						return errors.New("flexvolume add-ons not currently supported on coreos distro. Please use Ubuntu")
 					}
+					if common.IsKubernetesVersionGe(a.OrchestratorProfile.OrchestratorVersion, "1.16.0") {
+						log.Warnf("%s add-on will be DEPRECATED in favor of csi-secrets-store addon for 1.16+", addon.Name)
```

Comment:
So if user enables this and default creates secrets csi, then that means they will have both solutions running in the cluster?